### DISCUSSION TITLE
Question about how to open a new window - Preferences for example

### DISCUSSION THREAD

**Discussion by Mafi78** [jooy2/vutron#162] [Q&A] [answered]
Hi,
I love Vutron and i am thinking of developing an app with it. My plan is to have a preference page that should show up as a Modal window. I tried to include an 
`<a target="uniquename" href="/second"> `
but the only thing that happens it that i get an unresponse main app ? 
I am pretty sure i have to open up a new BrowserWindow Object from Vue- but i don'T figure it out ? Can somebpdy help me ?
BR Manfred

**Comment by jooy2** [comment, marked as answer]
Hello. Thank you for using the Vutron template.
In projects consisting of Vue+Electron, we prefer URLs consisting of hashes, so we use a hash-based router in Vutron as well.

References: https://router.vuejs.org/guide/essentials/history-mode.html#Hash-Mode

When navigating to a page, be sure to use a hash value, not a regular URL.
If you want to navigate to the page `/second`: Use `/#/second` or `/#second` instead of `/second`.

If you found this helpful, please leave a star on the GitHub project page - we appreciate your support!

**Comment by Mafi78** [comment]
Thank you very much !!
That worked like a charm :)
